File: C1.UWP.FlexChart/CS/FlexRadarIntro/EnumConverter.cs
using C1.Chart;
using System;
using Windows.UI.Xaml.Data;

namespace FlexRadarIntro
{
    public class EnumConverter : IValueConverter
    {
        public object Convert(object value, Type targetType, object parameter, string language)
        {
            return value.ToString();
        }

        public object ConvertBack(object value, Type targetType, object parameter, string language)
        {
            return Enum.Parse(targetType, value.ToString());
        }
    }
}

File: C1.UWP.FlexChart/VB/FlexRadarIntro/EnumConverter.vb
Imports C1.Chart

Public Class EnumConverter
    Implements IValueConverter
    Public Function Convert(value As Object, targetType As Type, parameter As Object, language As String) As Object Implements IValueConverter.Convert
        Return value.ToString()
    End Function

    Public Function ConvertBack(value As Object, targetType As Type, parameter As Object, language As String) As Object Implements IValueConverter.ConvertBack
        Return CType([Enum].Parse(targetType, value.ToString()), Palette)
    End Function
End Class